taichi/codegen/llvm/codegen_llvm.cpp

```diff
@@ -688,6 +688,11 @@ llvm::Type *TaskCodeGenLLVM::llvm_type(DataType dt) {
     return llvm::Type::getDoubleTy(*llvm_context);
   } else if (dt->is_primitive(PrimitiveTypeID::f16)) {
     return llvm::Type::getHalfTy(*llvm_context);
+  } else if (dt->is<TensorType>()) {
+    auto tensor_type = dt->cast<TensorType>();
+    auto element_type = llvm_type(tensor_type->get_element_type());
+    return llvm::VectorType::get(element_type, tensor_type->get_num_elements(),
+                                 false);
```

Comment:
Better add a comment to the argument, so it doesn't seem like a magic number:

llvm::VectorType::get(element_type, tensor_type->get_num_elements(), false /\*scalable\*/);